### DISCUSSION TITLE
Hi! Can someone help me turn on the side bar?

### DISCUSSION THREAD

**Discussion by lilyli142** [Godiesc/firefox-gx#228] [Q&A] [answered]
Hi! I am new to this, I don't know anything about coding, I'm just trying to put this extension on, I've manage to do everything that I want the only problem is the sidebar, I literally just don't know what to do, I've read all the thing inside the README, but I have no clue what to really do, I would be grateful if someone could help me, please and thank you :)

**Comment by Godiesc** [comment, marked as answer]
hi,
sorry for the late response, Github didn't send me the email notification, in the readme you have the preference you should create in `about:config` page and you should set your bookmarks toolbar to 'always show', Here is a [user that got some help](https://www.reddit.com/r/FirefoxCSS/comments/1gbz73j/need_help_fixing_the_left_sidebar_in_firefox_gx/) for that, and here's a video to show you how to create the preference: https://imgur.com/a/H1C6dKv

**Reply by lilyli142** [reply]
It worked! I just did it, thank you so much :)
